### ISSUE TITLE
System Settings' header is overlapping the submenu sidebar

### ISSUE THREAD

**Issue by nghtamm2003** [Kalyvara/NXTheme-CHROMA#4]
Firmware 18.1.0, Atmosphere 1.7.1
![PXL_20240706_133034954](https://github.com/Kalyvara/NXTheme-CHROMA/assets/35840145/f43334ce-8e4b-403f-8e18-f5b0c53c075e)

**Comment by Kalyvara** [comment]
I'm not sure I understand the issue here. This is normal behavior, the header spans horizontally and will always come in front of the other panes.

**Comment by nghtamm2003** [comment]
Yes, but the header is currently overlapping even the first submenu in the sidebar (as in the picture). Not really a big problem but basically, no one can't see what the first submenu is.

**Comment by nghtamm2003** [comment]
Sorry, because English isn't my mother tongue, my words may not be very clear. It's defaulted to Airplane Mode but I can only see a small highlighted padding on the sidebar, as shown in the picture.

**Comment by nghtamm2003** [comment]
https://youtu.be/2CPtkIKKyU0
Let's take a look about what I said over here!

**Comment by Kalyvara** [comment]
Weird, Nintendo must have changed the Settings' layout or something. Worked fine on 16.x. I'll see what I can do, thanks for reporting.

**Comment by nghtamm2003** [comment]
Thanks! I was trying to fix it but I'm not used to these themes' JSON so I opened this issue, but it's no big deal so no rush.
